newswires/app/AppComponents.scala

```diff
@@ -46,8 +46,11 @@ class AppComponents(context: Context)
     .build()
 
   if (context.environment.mode == Mode.Dev) {
-    // TODO run against a local DB?
-    Database.configureRemoteDevDb(ssmClient)
+    if (sys.env.contains("USE_CODE_DB")) {
```

Comment:
not super happy about using an environment variable here

Reply:
we discussed in-person and ideally we might instead have an environment variable more specific to connect differences, such as `USE_IAM_AUTH` but let's save for a future PR